hbase-server/src/test/java/org/apache/hadoop/hbase/replication/TestReplicationEndpoint.java

```diff
@@ -396,11 +447,11 @@ public void testMetricsSourceBaseSourcePassThrough() {
     // after calling #setAgeOfLastShippedOpByTable
     boolean containsRandomNewTable = source.getSingleSourceSourceByTable()
         .containsKey("RandomNewTable");
-    Assert.assertEquals(false, containsRandomNewTable);
+    assertEquals(false, containsRandomNewTable);
     source.updateTableLevelMetrics(createWALEntriesWithSize("RandomNewTable"));
     containsRandomNewTable = source.getSingleSourceSourceByTable()
         .containsKey("RandomNewTable");
-    Assert.assertEquals(true, containsRandomNewTable);
+    assertEquals(true, containsRandomNewTable);
```

Comment:
nit: `assertTrue(containsRandomNewTable)`

Reply:
Not originally the scope of this PR, I'm reverting this change...